charts/tenant-base/chart/templates/configmap.yaml

```
@@ -0,0 +1,9 @@
+{{- range .Values.configMaps }}
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ printf "%s-%s" (include "chart.fullname" $) .name }}
+data:
+{{ .content | toYaml | indent 2 }}
```

A bit easier to understand if it is indented:
```suggestion
  {{- .content | toYaml | nindent 2 }}
```